embedchain/llm/base.py

```diff
@@ -74,6 +77,19 @@ def _format_history(self) -> str:
         """
         return "\n".join(self.history)
 
+    def _format_memories(self, memories: list[dict]) -> str:
+        """Format memories to be used in prompt
+
+        :param memories: Memories to format
+        :type memories: list[dict]
+        :return: Formatted memories
+        :rtype: str
+        """
+        formatted_memories = []
+        for memory in memories:
+            formatted_memories.append(f"Question: {memory['input']}\nAnswer: {memory['text']}")
```

Comment:
Not sure if I understand why you need to fetch input and text. 

`input` is the text that user entered to create the memory
`text` is the actual memory that got created from the user input.

You probably only want to pass the `text` unless it is intentional.

Reply:
I did because the format of conversation history and memory would be consistent. But here it makes sense to only use `text` as it is generated by mem0, I'll make that change.